src/layouts/EditHomepage/constants.ts

```diff
@@ -3,6 +3,27 @@ export const RESOURCES_SECTION = {
   subtitle: "Add a preview and link to your Resource Room",
   id: "resources",
 } as const
+export const LocalDateTimeNow = new Date()
```

Comment:
nit: either lowercase this or screaming snake (more inclined to  prev) 

**potential bug**: this seems to only construct the date **once** - ie, if i stay on this page and call this `const` again, it might not update. perhaps this should be a function instead?

Reply:
you are right, c1bb6644bb42dc6e734bf522ec550a434e4f1750